packages/appstore/src/components/wallets-banner/wallets-banner.scss

```diff
@@ -0,0 +1,185 @@
+        border-radius: $BORDER_RADIUS * 4;
+
+        @include mobile {
+            padding: 1.4rem 1.6rem;
+            border-radius: $BORDER_RADIUS * 2;
+        }
+    }
+
+    &__image {
+        position: absolute;
+        right: 0;
+        bottom: 0;
+        border-top-right-radius: $BORDER_RADIUS * 4;
+        border-bottom-right-radius: $BORDER_RADIUS * 4;
+
+        @include mobile {
+            border-top-right-radius: $BORDER_RADIUS * 2;
+            border-bottom-right-radius: $BORDER_RADIUS * 2;
+        }
+    }
+
+    &__upgrade-banner {
+        height: 180px;
+
+        @include mobile {
+            height: 108px;
+        }
+
+        &-description {
+            width: 45%;
+            height: 100%;
+            display: flex;
+            flex-direction: column;
+            justify-content: center;
+            align-items: baseline;
+
+            @include mobile {
+                width: 70%;
+            }
+        }
+
+        &-button {
+            margin-top: 0.8rem;
+            padding-inline: 6.7rem;
+
+            @include mobile {
+                padding-inline: 1.6rem;
+            }
+        }
+    }
+
+    &__upgrading-banner {
+        display: flex;
+        align-items: center;
+        height: 182px;
+
+        @include mobile {
+            height: 142px;
+        }
+
+        &-description {
+            width: 45%;
+            height: 80%;
+            display: flex;
+            flex-direction: column;
+            justify-content: space-around;
+            align-items: baseline;
+
+            @include mobile {
+                width: 70%;
+            }
+        }
+
+        &-loading {
+            span {
+                animation-name: loading;
+                animation-duration: 1.5s;
+                animation-iteration-count: infinite;
+                animation-timing-function: cubic-bezier(1, 0, 0, 1);
+            }
+
+            span:nth-child(2) {
+                animation-delay: 0.5s;
+            }
+
+            span:nth-child(3) {
+                animation-delay: 1s;
+            }
+        }
+
+        &-dot {
+            height: 0.8rem;
+            width: 0.8rem;
+            background-color: $color-grey-6;
+            border-radius: 50%;
+            display: inline-block;
+            margin-right: 0.6rem;
+
+            @include mobile {
+                height: 0.5rem;
+                width: 0.5rem;
+            }
+        }
+
+        &-image {
+            opacity: 50%;
+            height: 100%;
+        }
+    }
+
+    &__ready-banner {
+        display: flex;
+        align-items: center;
+        height: 230px;
+        background-color: #e2f3f3;
+
+        @include mobile {
+            height: 190px;
+        }
+
+        &-tick {
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            background-color: #4ab4b3;
```

Comment:
same var(--)

Reply:
@thisyahlen-deriv I just want to know why is it better to use var instead of constant?